### PR TITLE
Kirilushkin Aleksandr Tasks 11-20

### PR DESCRIPTION
**Pull request by KirilushkinAleksandr**

### CODE REVIEW

File: tasks/12_title-case/title-case.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+function titleCase(str) {
+  let arr = str.split(' ');
+  arr = arr.map((item) =>
```
Comment (alexkond) [inline tasks/12_title-case/title-case.js]: It is a bad practice to rewrite array variable, instead, you could return a new array, produced by the map and joined.

**Review by alexkond — CHANGES_REQUESTED** [review]

File: tasks/15_confirm-ending/confirm-ending.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+function confirmEnding(str, target) {
+  return (str.indexOf(target, str.length - target.length) === -1)
```
Comment (alexkond) [inline tasks/15_confirm-ending/confirm-ending.js]: Doesn't this expression returns `boolean` itself? Why you're using ternary operator here?

**Review by alexkond — CHANGES_REQUESTED** [review]

**Review by alexkond — DISMISSED** [review]

File: tasks/11_find-longest-word/find-longest-word.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+function findLongestWord(str) {
+  const words = str.split(' ');
+  return words.reduce(function(longest, current) {
+    if (current.length > longest) {
+      return current.length;
```
Comment (KalachevDev) [inline tasks/11_find-longest-word/find-longest-word.js]: Take a look at Math.max

File: tasks/18_humanize-format/humanize-format.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+function humanizeFormat(number) {
+  if (!number) {
```
Comment (KalachevDev) [inline tasks/18_humanize-format/humanize-format.js]: In this case for 0 you will return empty string as well

**Review by KalachevDev — CHANGES_REQUESTED** [review]

**Review by vovcyan — DISMISSED** [review]

File: tasks/18_humanize-format/humanize-format.js (hunk 4 above)
Comment (KirilushkinAleksandr) [reply tasks/18_humanize-format/humanize-format.js]: Despite such case isn't mentioned in tech task, fixed

**Review by vovcyan — DISMISSED** [review]

**Review by alexkond — DISMISSED** [review]

**Review by vovcyan — APPROVED** [review]

**Review by alexkond — APPROVED** [review]

**Pull request merged**